src/abstractitensornetwork.jl

```diff
@@ -870,6 +870,44 @@ function ITensors.commoninds(tn1::AbstractITensorNetwork, tn2::AbstractITensorNe
   return inds
 end
 
+"""Add two itensornetworks together by growing the bond dimension. The network structures need to be have the same vertex names, same site index on each vertex """
+function add_itensornetworks(tn1::AbstractITensorNetwork, tn2::AbstractITensorNetwork)
```

Comment:
We could just call this `add(...)`, and then also provide an alias `+(tn1::AbstractITensorNetwork, tn2::AbstractITensorNetwork) = add(tn1, tn2)` (where `+` would be imported from `Base`.

Reply:
Note that ITensors.jl exports `add` so probably we also want to import `add` from ITensors.

Reply:
Okay cool I believe I did this.